perun-core/src/main/java/cz/metacentrum/perun/core/provisioning/HierarchicalHashedDataGenerator.java

```diff
@@ -72,7 +73,7 @@ public HashedGenData generateData() {
 		List<Resource> resources =
 				sess.getPerunBl().getFacilitiesManagerBl().getAssignedResources(sess, facility, null, service);
 
-		if (BeansUtils.getCoreConfig().getForceConsents()) {
+		if (consentEval && BeansUtils.getCoreConfig().getForceConsents()) {
```

Comment:
we\ll want to filter out members that do not have consent even if calling it without the consentEval flag, we just want to skip creation of new consents in that case